flask_rebar/utils/request_utils.py

```diff
@@ -110,28 +142,30 @@ def normalize_schema(schema):
     return schema
 
 
-def raise_400_for_marshmallow_errors(errs, msg):
+def raise_400_for_marshmallow_errors(
+    errs: Dict[str, Any], msg: Union[str, messages.ErrorMessage]
+) -> errors.BadRequest:
     """
     Throws a 400 error properly formatted from the given marshmallow errors.
 
-    :param dict errs: Error dictionary as returned by marshmallow
+    :param dict: Error dictionary as returned by marshmallow
     :param Union[str,messages.ErrorMessage] msg: The overall message to use in the response.
     :raises: errors.BadRequest
     """
     if not errs:
-        return
+        return errors.BadRequest(msg=msg)
 
     copied = copy.deepcopy(errs)
 
     _format_marshmallow_errors_for_response_in_place(copied)
 
     additional_data = {"errors": copied}
-    message, _ = msg if isinstance(msg, tuple) else msg, None
+    message, _ = msg if isinstance(msg, tuple) else (msg, None)
```

Comment:
I think this was a typo - the original code was basically equivalent to `message, _ = (msg if isinstance(msg, tuple) else msg), None` which is identical to `message = msg` and I don't think that was the intent.